client/src/app/auth/register/register.component.html

```diff
@@ -0,0 +1,21 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
There are same imports in login.component.html - if they are necessary in both locations try to move it to some common place, so it wouldn't be downloaded again (i.e. app.component.html)